### ISSUE TITLE
Release 3.0.1 is broken for default composer installs

### ISSUE THREAD

**Issue by jakedaleweb** [ralouphie/getallheaders#7]
As the `composer.json` has been set to `export-ignore` in `.gitattributes` it is not included when this repo is pulled in with a default `composer install` command. this leads to the following Exception being thrown:

`Could not read /path/to/vendor/ralouphie/getallheaders/composer.json`

To get around this anyone installing this module via composer must specify `--prefer-source` in their install command.

**Comment by ralouphie** [comment]
Thanks for bringing this to my attention. This PR should fix the issue:
https://github.com/ralouphie/getallheaders/pull/8

This change has been merged and is included in `3.0.2`.

**Comment by legoktm** [comment]
@jakedaleweb Can you provide a little more information about the error you're getting? What version of composer are you using? Do you have any plugins installed? I can't reproduce your error:
```
km@km-pt /tmp> composer --version
Composer version 1.8.0 2018-12-03 10:31:16
km@km-pt /tmp> cat composer.json 
{
    "require": {
        "ralouphie/getallheaders": "3.0.1"
    }
}
km@km-pt /tmp> composer install
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing ralouphie/getallheaders (3.0.1): Downloading (100%)         
Writing lock file
Generating autoload files
km@km-pt /tmp> ls vendor/ralouphie/getallheaders/
LICENSE  README.md  src/
```

**Comment by jakedaleweb** [comment]
@legoktm 

The issue appeared during some CI steps. We are using this Dockerfile which installs composer https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/php/images/7.1.15-apache-jessie/Dockerfile

The installed version is as follows:

```
$ composer --version
Composer version 1.7.3 2018-11-01 10:05:06
```

The fix in `3.0.2` resolved the issue for us.
